packages/calcite-components/src/components/shell-center-row/shell-center-row.e2e.ts

```diff
@@ -92,3 +94,31 @@ describe("calcite-shell-center-row", () => {
     `);
   });
 });
+
+describe("theme", () => {
+  const shellCenterHtml = html`
+    <calcite-shell-center-row detached slot="center-row">
+      <calcite-action-bar class="calcite-mode-dark" slot="action-bar">
+        <calcite-action-group>
+          <calcite-action text="Add" label="Add Item" icon="plus"></calcite-action>
+          <calcite-action text="Save" label="Save Item" icon="save"></calcite-action>
+        </calcite-action-group>
+      </calcite-action-bar>
+    </calcite-shell-center-row>
+  `;
+  describe("default", () => {
+    const tokens: ComponentTestTokens = {
+      "--calcite-shell-center-row-border-color": {
```

Comment:
Can you move this to a separate group for slotted overrides? Similar to your [`shell` test](https://github.com/Esri/calcite-design-system/pull/9458/files#diff-068948d3f9d227bd434c56db88362d26396096d7be28e1845ef28b53c6c0f83fR126).